go/vm/test/world_state.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package vm_test
+
+import "github.com/Fantom-foundation/Tosca/go/vm"
+
+type WorldState map[vm.Address]Account
+
+//var _ vm.WorldState = &WorldState{}
```

Comment:
commented code ?

Reply:
I removed this file from the PR, it was not intended to be there, sorry.